lib/apiservers/engine/backends/container.go

```diff
@@ -1206,14 +1209,21 @@ func (c *ContainerBackend) findPortBoundNetworkEndpoint(op trace.Operation, host
 	return nil, nil
 }
 
-// ContainerStop looks for the given container and terminates it,
+// ContainerStop, external entry point
+func (c *ContainerBackend) ContainerStop(name string, seconds *int) error {
+	op := trace.NewOperation(context.Background(), "ContainerStop: %s", name)
+	defer trace.End(trace.Audit(name, op))
```

Comment:
There are twice of defer trace.End for this operation. Is that OK?

Reply:
It lets us differentiate between `containerStop` called via `ContainerStop` and `ContainerRm`. In a debug configuration that's definitely worthwhile.